updater/update_firmware.py

```diff
@@ -10,7 +10,27 @@
 import spsdk.sdp.interfaces.usb as sdp_usb
 import spsdk.mboot.interfaces.usb as mboot_usb
 from spsdk.mboot import McuBoot
-
+import usb
+from typing import NamedTuple
+
+class FirmwareInfo(NamedTuple):
+    tag: str
+    branch: str
+    commit: str
+    board: str
+
+def get_firmware_info():
+    dev = usb.core.find(idVendor=0x16d0, idProduct=0x10a7)
+    if dev is None:
+        print('No DUO connected.')
+        return None
+    else:
+        return FirmwareInfo(
+                tag = usb.util.get_string(dev, 4),
+                branch = usb.util.get_string(dev, 5),
+                commit = usb.util.get_string(dev, 6),
+                board = usb.util.get_string(dev, 7))
```

Comment:
We can just have `from firmware_info import get_firmware_info`, rather than copying things into the updater script.

Reply:
Added in [6d1669b](https://github.com/datomusic/duo-imxrt/pull/134/commits/6d1669bf1598fca5096f856a2fa9b036787ad699)